src/lib/viewers/doc/_docBase.scss

```diff
@@ -3,8 +3,19 @@
 .bp {
     .bp-thumbnails-container {
         border-right: solid 1px $seesee;
+        display: flex;
         flex: 0 0 180px;
     }
+
+    .bp-thumbnail {
```

Comment:
You may want to look into adding an `object-fit` rule when images come into play here, depending on what the design looks like.